### PR TITLE
refactoring with prettier

### PR DESCRIPTION
**Pull request by alaninfo23**
- Adding test to verify that when we insert a task with an empty title, the user should not be able to see the text 'Sed ut perspiciatis unde...' in the task details.

- A small refactoring of the code was also done.

image

### CODE REVIEW

File: src/integration/tests/firstTest.test.jsx [hunk 1]
```diff
@@ -98,45 +98,62 @@ describe("Test project in the app task list.", () => {
...
+    userEvent.clear(input);
+    userEvent.type(input, taskMsg);
+    const addTask = screen.getByTestId("ADD_BUTTON");
+    userEvent.click(addTask);
+    const task = within(screen.getByTestId(`TASK_CONTAINER_${taskMsg}`));
+    const infoTask = task.getByTestId("INFO_TASK_BUTTON");
+    userEvent.click(infoTask);
+
+    const details = screen.getByTestId("TASK_TITLE");
+    expect(details).toHaveTextContent(
```
Comment (norikoStefany-daitan) [inline src/integration/tests/firstTest.test.jsx:114]: Shouldn't it be .not.toHaveTextContent?

```suggestion
    expect(details).not.toHaveTextContent(
```

**Review by norikoStefany-daitan — APPROVED** [review]
